.github/workflows/coding.yml

```
@@ -0,0 +1,58 @@
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: npm ci
+        env:
+          TEST: true
+      - name: Check code
+        run: ls -all
+      # - name: Lint code  
+      #  run: npm run lint  # Assuming you have a lint script in your package.json  
+  
+  # TEST
+  test-scala-coding:  
```

yamllint

[error] trailing spaces

(trailing-spaces)